### PR TITLE
DOCSP 37460 Updating Compass Multiple Connections Screenshots

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Updating Compass screenshots to show the new multiple connections UI and deleting unused images. 

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-37460/connect/connections/
https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-37460/databases/
https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-37460/collections/
https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-37460/connect/advanced-connection-options/general-connection/#procedure

## JIRA

https://jira.mongodb.org/browse/DOCSP-37460

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66bcd07383839571170e7f86

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/includes/open-new-connection.rst [hunk 1]
```diff
@@ -1,3 +1,7 @@
-In the bottom panel of the :guilabel:`Connections Sidebar`,
-click the :icon-lg:`Plus` icon to open the :guilabel:`New Connection`
-modal. 
\ No newline at end of file
+In the bottom panel of the :guilabel:`Connections Sidebar`, click
```
Comment (ianf-mongodb) [inline source/includes/open-new-connection.rst:1]: *Question* 

Is this file included anywhere? I did a quick search in master and dont see it included and dont see it in the PR, could we also remove this?

Screenshot 2024-08-14 at 2 09 39 PM

**Review by ianf-mongodb — APPROVED** [review]
Hi @ltran-mdb2, this LGTM %2 nits, I think 1 more file can be removed and would suggest removing one of the red circles from 1 of the images.

File: source/includes/open-new-connection.rst (hunk 1 above)
Comment (ltran-mdb2) [reply source/includes/open-new-connection.rst:1]: Done!

File: source/images/compass/databases-view.png
[the remark this answers is not in the document]
Comment (ltran-mdb2) [reply source/images/compass/databases-view.png:1]: Done!

File: source/includes/open-new-connection.rst (hunk 1 above)
Comment (ltran-mdb2) [reply source/includes/open-new-connection.rst:1]: This file is added in the multiple-connections branch and is used in the advanced connection options pages (just added the staging link). It's not currently on master but will be once the feature branch is merged.

**Review by lerouxb — APPROVED** [review]

**Pull request merged**